src/pizzeria/Order.java

```diff
@@ -0,0 +1,107 @@
+package pizzeria;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Random;
+
+public class Order {
+
+
+    private int orderNumber;
+    private int clientNumber;
```

Comment:
1. Create separate Customer class and move its related fields there.
2. Create separate Ingredient class
3. Create separate Pizza class and move its related fields there.
4. Create separate OrderItem class